app/src/main/java/no/usn/mob3000/ui/components/socials/friendrequests/friendRequestItem.kt

```diff
@@ -0,0 +1,115 @@
+package no.usn.mob3000.ui.components.socials.friendrequests
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import no.usn.mob3000.R
+import no.usn.mob3000.domain.model.auth.UserProfile
+import no.usn.mob3000.domain.model.social.FriendRequestData
+import no.usn.mob3000.ui.theme.DefaultButton
+
+/**
+ * Function that displays a single friend request item in the list.
+ *
+ * This item shows the user who sent the friend request, along with "Accept" and "Decline" buttons
+ * to allow the user to respond to the request.
+ *
+ * @author Husseinabdulameer11
```

Comment:
Incorrectly placed author KDoc property. Should come after parameter KDoc properties, and also should have a date property.